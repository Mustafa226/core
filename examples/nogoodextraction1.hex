node(a).
node(b).
node(c).
node(d).
node(e).
node(f).
node(g).

in(X) v out(X) :- node(X).
c(X) :- &id[in](X).  % &nonmonid is an identity function but declared to be nonmonotonic
:- &countext[c](C), C > 2.
