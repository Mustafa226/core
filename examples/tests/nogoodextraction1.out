{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(c),out(d),out(e),out(f),out(g),in(a),in(b),c(a),c(b)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(b),out(d),out(e),out(f),out(g),in(a),in(c),c(a),c(c)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(b),out(c),out(e),out(f),out(g),in(a),in(d),c(a),c(d)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(b),out(c),out(d),out(f),out(g),in(a),in(e),c(a),c(e)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(b),out(c),out(d),out(e),out(g),in(a),in(f),c(a),c(f)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(b),out(c),out(d),out(e),out(f),in(a),in(g),c(a),c(g)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(b),out(c),out(d),out(e),out(f),out(g),in(a),c(a)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(d),out(e),out(f),out(g),in(b),in(c),c(b),c(c)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(c),out(e),out(f),out(g),in(b),in(d),c(b),c(d)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(c),out(d),out(f),out(g),in(b),in(e),c(b),c(e)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(c),out(d),out(e),out(g),in(b),in(f),c(b),c(f)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(c),out(d),out(e),out(f),in(b),in(g),c(b),c(g)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(c),out(d),out(e),out(f),out(g),in(b),c(b)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(e),out(f),out(g),in(c),in(d),c(c),c(d)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(d),out(f),out(g),in(c),in(e),c(c),c(e)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(d),out(e),out(g),in(c),in(f),c(c),c(f)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(d),out(e),out(f),in(c),in(g),c(c),c(g)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(d),out(e),out(f),out(g),in(c),c(c)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(f),out(g),in(d),in(e),c(d),c(e)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(e),out(g),in(d),in(f),c(d),c(f)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(e),out(f),in(d),in(g),c(d),c(g)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(e),out(f),out(g),in(d),c(d)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(d),out(g),in(e),in(f),c(e),c(f)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(d),out(f),in(e),in(g),c(e),c(g)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(d),out(f),out(g),in(e),c(e)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(d),out(e),in(f),in(g),c(f),c(g)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(d),out(e),out(g),in(f),c(f)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(d),out(e),out(f),in(g),c(g)}
{node(a),node(b),node(c),node(d),node(e),node(f),node(g),out(a),out(b),out(c),out(d),out(e),out(f),out(g)}
